### ISSUE TITLE
Figures displayed in LaTeX

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco#1502]
The change in `ppfig.save_figure` has messed up the display in the latex templates. 

Right now, including the figures as they are with `\includegraphics` without `trim` or `clip` seems to be (and should be) just fine. IMHO the very specific trimming and clipping like
```
\includegraphics[width=0.253\textwidth, trim= 0.7cm 0.8cm 0.5cm 0.5cm, clip]{ppfigs_f017}&
\includegraphics[width=0.238\textwidth, trim= 1.8cm 0.8cm 0.5cm 0.5cm, clip]{ppfigs_f018}&
\includegraphics[width=0.238\textwidth, trim= 1.8cm 0.8cm 0.5cm 0.5cm, clip]{ppfigs_f019}&
\includegraphics[width=0.238\textwidth, trim= 1.8cm 0.8cm 0.5cm 0.5cm, clip]{ppfigs_f020}\\
\includegraphics[width=0.253\textwidth, trim= 0.7cm 0.0cm 0.5cm 0.5cm, clip]{ppfigs_f021}&
\includegraphics[width=0.238\textwidth, trim= 1.8cm 0.0cm 0.5cm 0.5cm, clip]{ppfigs_f022}&
\includegraphics[width=0.238\textwidth, trim= 1.8cm 0.0cm 0.5cm 0.5cm, clip]{ppfigs_f023}&
\includegraphics[width=0.238\textwidth, trim= 1.8cm 0.0cm 0.5cm 0.5cm, clip]{ppfigs_f024}
```
wasn't a promising and long-lasting approach in the first place.

Commit dc2a2510404201d37b0c961dc56ff411b70384c7 is the last commit which should work with the current latex templates, commit 5b6ddcb31336826ad7af8e7372554c892740fab0 is the (first) commit breaking the templates. Calling ``save_figure(..., layout_rect=None, bbox_inches='tight')`` since commit 47fb9e4b9353457d48aeac9b076df3374937ae3e should restore the original behavior (not tested though). 

EDIT: the scatterplots have changed in that the title is now shown as title (as it should) and not as ylabel.

**Comment by brockho** [comment]
I am assigning myself to this issue as I anyway wanted to have a look at the LaTeX templates related to #1404.

**Comment by nikohansen** [comment]
IMHO, unless there is a good reason we should produce figures which can be included without trimming or clipping. I don't know whether we should keep the hand-made labels to the ECDF graphs. If we drop them it forces the user to the get the algorithm names right within the post-processing.

**Comment by nikohansen** [comment]
Re. labels in ECDF graphs: when we are going to switch away from `matplotlib`, the way how to annotate these graphs may change as well, so latex hand-made labels may again break.

**Comment by brockho** [comment]
Decision of yesterday's discussion: changing the ECDF algorithms labels seems handy for some of us, hence, we try to keep it for now. Other trimming and clipping, however, should go away. I'll take care of this.

**Comment by brockho** [comment]
Progress report in the `dimo-latex-issue1502` branch (in my fork):
* [X] bbob 1 algo
* [X] bbob 2 algo
* [X] bbob 3+ algo
* [ ] bbob-biobj 1 algo
* [ ] bbob-biobj 2+ algo
* [X] bbob-noisy

**Comment by brockho** [comment]
The merged pull request #1661 now addresses all the possible combinations of test suite and number of algorithms (I hope) such that the LaTeX templates look again right with the newest matplotlib version.
